r_sort.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+from merge_sort import timed_func
+
+
+@timed_func
+def r_sort(sequence):
+    digit = 1
+    while True:
+        bins = [[] for i in range(10)]
+        for x in sequence:
+            # the expression in bins returns the number in digit place
+            bins[x % 10 ** digit // 10 ** (digit -1)].append(x)
```

Comment:
This works, but it has both a modulus operation and a floored division operation.  That's going to be pretty slow.  Can you find a more efficient operation that would have the same result?